e2e-tests/README.md

```diff
@@ -0,0 +1,109 @@
+# End - End UI Tests
+### Overview
+The End - End UI framework is based on [Cucumber](https://github.com/cucumber-rs/cucumber "Cucumber") and [Thrityfour](https://github.com/stevepryde/thirtyfour/tree/main/thirtyfour "Thrityfour"). The tests are written in Gherkin format and they are executed with the aid of cucumber-rs. 
+
+The tests under package e2e-tests can be run with the below command,
+```shell
+git clone https://github.com/trustification/trustification
+cd e2e-tests 
+cargo run -- --application http://localhost:8084/ --user-name admin --password admin123456
```

Comment:
I think you should add a note about how to deploy what's running on `localhost:8084`:

```suggestion
git clone https://github.com/trustification/trustification
# in another shell...
cd trustification/deploy/compose/
podman-compose -f compose.yaml -f compose-guac.yaml -f compose-trustification.yaml up
# ...back to original shell
cd trustification/e2e-tests 
cargo run -- --application http://localhost:8084/ --user-name admin --password admin123456
```

Reply:
@jcrossley3 updated! Please let me know :)